### PR TITLE
READY FOR REVIEW - Fix tests (CI is fixed in #93)

### PR DESCRIPTION
**Pull request by posita**
- Fix broken tests.
- Additional repo cleanup.

FWIW, it looks like tests have been broken since at latest 76536ae6. 😳

### CODE REVIEW

**Comment by posita** [comment]
CI is fixed in #93, but this PR is designed to be independent of that one (i.e., not impose an order of acceptance).

File: test/test_dropbox.py [hunk 1]
```diff
@@ -69,41 +73,42 @@ def test_bad_auth(self):
         invalid_token_dbx = Dropbox(INVALID_TOKEN)
         with self.assertRaises(AuthError) as cm:
             invalid_token_dbx.files_list_folder('')
-        self.assertEqual(cm.exception.error['error']['.tag'],
-                         'invalid_access_token')
+        self.assertEqual(cm.exception.error._tag, 'invalid_access_token')
```
Comment (posita) [inline test/test_dropbox.py]: There _has_ to be a better way to check this, but I can't find one. 😞

Comment (posita) [inline test/test_dropbox.py]: @braincore, any ideas?

Comment (braincore) [inline test/test_dropbox.py]: `cm.exception.error.is_invalid_access_token()`

File: README.rst [hunk 2]
```diff
@@ -38,22 +37,21 @@ Installation
...
+::
 
     $ git clone git://github.com/dropbox/dropbox-sdk-python.git
     $ cd dropbox-sdk-python
     $ python setup.py install
 
 After installation, to get started, open a Python console:
 
-.. code-block:: python
+::
```
Comment (braincore) [inline README.rst]: Does it auto detect the language? I thought giving it a suggestion improved the syntax highlighting.

File: test/test_dropbox.py (hunk 1 above)
Comment (posita) [inline test/test_dropbox.py]: _smacks forehead._

File: README.rst (hunk 2 above)
Comment (posita) [inline README.rst]: No, but the `code-block` syntax is a Sphinx extension that's not available on PyPi. PyPi will botch the formatting on the package page (and will likely screw up parsing the classifiers) if these are included in the `long_description`.

Comment (posita) [inline README.rst]: Okay, I see the problem. They had the wrong parsers. Fixed now.

File: test/test_dropbox.py (hunk 1 above)
Comment (posita) [inline test/test_dropbox.py]: Fixed now.

**Pull request merged**